xtask/src/docgen.rs

```diff
@@ -48,6 +54,67 @@ pub fn typable_commands() -> Result<String, DynError> {
     Ok(md)
 }
 
+pub fn static_commands() -> Result<String, DynError> {
+    let mut md = String::new();
+    let keymaps = [
+        EditorView::default().keymaps.map()[&Mode::Normal].reverse_map(),
+        EditorView::default().keymaps.map()[&Mode::Select].reverse_map(),
+        EditorView::default().keymaps.map()[&Mode::Insert].reverse_map(),
+    ];
+
+    md.push_str(&md_table_heading(&[
+        "Name".to_owned(),
+        "Description".to_owned(),
+        "Default keymaps".to_owned(),
+    ]));
+
+    for cmd in MappableCommand::STATIC_COMMAND_LIST {
+        let keymap_strings: Vec<String> = keymaps
+            .iter()
+            .enumerate()
+            .map(|(_, keymap)| {
+                keymap
+                    .get(cmd.name())
+                    .map(|bindings| {
+                        bindings.iter().fold(String::new(), |mut acc, bind| {
+                            if !acc.is_empty() {
+                                acc.push_str(", ");
+                            }
+                            for key in bind {
+                                acc.push_str(&key.key_sequence_format());
+                            }
+                            acc
+                        })
+                    })
+                    .unwrap_or_default()
+            })
+            .collect();
+
+        let keymap_string = keymap_strings
+            .iter()
+            .filter(|s| !s.is_empty())
+            .map(|s| {
+                let mode = match keymap_strings.iter().position(|t| t == s) {
+                    Some(0) => "normal",
+                    Some(1) => "select",
+                    Some(2) => "insert",
+                    _ => unreachable!(),
+                };
+                format!("[{}] ({})", s, mode)
```

Comment:
for the display here I think it would be clearer to say the mode first. The `[...]` notation is also a special Markdown syntax that would be nice to avoid.

For example

| Name | Description | Default keymaps |
| --- | --- | --- |
| `no_op` | Do nothing |  |
| `move_char_left` | Move left | normal: `` <left> ``, `` h ``, insert: `` <left> `` |
| `move_char_right` | Move right | normal: `` <right> ``, `` l ``, insert: `` <right> `` |